core/bones/base.py

```diff
@@ -914,4 +1018,9 @@ def structure(self) -> dict:
             }
         else:
             ret["multiple"] = self.multiple
+        if self.compute:
+            ret["compute"] = {"method": self.compute.threshold.method.name}
+            if self.compute.threshold.lifetime:
+                ret["compute"]["lifetime"] = str(self.compute.threshold.lifetime)
+                ret["compute"]["last_updated"] = self.compute.threshold.last_updated.strftime("%Y-%m-%dT%H-%M-%S")
```

Comment:
Can we use `.isoformat()` here? 
```suggestion
                ret["compute"]["last_updated"] = self.compute.threshold.last_updated.isoformat()
```